tests/test_functions.py

```diff
@@ -872,3 +872,24 @@ def test_reset_index_inplace_drop(dataframe):
         dataframe.reset_index(drop=True),
         dataframe.reset_index_inplace(drop=True),
     )
+
+        
+def test_select_columns(dataframe):
+    df = dataframe.select_columns(columns= [
+        "a",
+        "Bell__Chart",
+        "cities"
+    ])
+    assert df is dataframe[[
```

Comment:
I think I see what the issue might be! If you look above, there's a few pandas testing functions that we use.

Here's a suggestion, taken from the [pandas testing utilities](https://pandas.pydata.org/pandas-docs/stable/generated/pandas.testing.assert_frame_equal.html):

```suggestion
    assert pd.testing.assert_frame_equal(df, dataframe[columns])
```